docs/billing.rst

```diff
@@ -0,0 +1,91 @@
+Cloud. Since - depending on your chosen offer - billing and payment forms may
+appear in different places across the CrateDB Cloud interface, this guide
+should function as a single reference document for finding the information you
+need about billing, invoicing, and payments.
+
+
+.. _billing-principles:
+
+Billing principles
+==================
+
+CrateDB Cloud has three general principles for billing. Firstly, we only ever
+bill actual usage of any of our services. This means there are no flat fees or
+minimum payments.
+
+Secondly, we only bill for a given period. This means any usage costs are
+rounded up to the nearest hour of use for Marketplace customers and to the
+nearest minute of use for customers directly deploying via the CrateDB Cloud
+Console.
+
+Thirdly, we measure usage based on Database Transaction Units (DTUs). These
+units represent a given combination of hardware capabilities and storage for a
+given subscription plan. They function as preconfigured usage units to make
+tracking your usage more convenient. This means the price you pay for any
+:ref:`subscription plan <gloss-subscription-plan>` is expressed as the cost of
+a given number of DTUs per minute or hour (as explained above). For more
+details about DTUs, see :ref:`our explanation of subscription plans
+<subscription-plans-dtus>`.
+
+
+.. _billing-info:
+
+Billing information
+===================
+
+Your billing information consists of your (company) address, credit card
+details, country of residence, VAT info, and so forth. This information can be
+filled out whenever you make use of an offer on CrateDB Cloud that is not free
+(when you only use a free offer, the billing info page will not be visible to
+you). There are several ways you can provide the necessary billing information:
+
+* As part of deploying a cluster for a new organization. If you deploy a
+  cluster that is not free, you will be prompted for your billing information
+  as part of the configuration wizard.
+* By using the Billing tab in the Organization overview of the CrateDB Cloud
+  Console. If you have a promoted cluster deployed and the free period expires,
+  you can find the Billing tab by going to the Organization overview and
+  clicking the fifth tab from the left. Here you can enter your billing details
+  and your billing method (credit card). See the :ref:`CrateDB Console
+  walkthrough <overview-org-billing>` for more information.
+
+
+.. _billing-invoicing:
+
+Invoicing
+=========
+
+Invoicing is handled variously depending on which deployment method you use.
+If you deploy your cluster directly via the CrateDB Cloud Console, you will be
+invoiced at the email address you provided on `signing up with CrateDB Cloud`_.
+If you use one of the marketplace offers, the invoicing is handled by the
+marketplace provider in question and will be part of your general invoicing for
+services via that marketplace.
+
+Currently, VAT charges for EU customers are handled by Crate.io directly and
```

Comment:
I believe VAT charges are added by AWS/Azure and not us when billed through them...